SchibstedAccount/UI/IdentityUIViewController.swift

```diff
@@ -66,7 +66,7 @@ class IdentityUIViewController: UIViewController {
         return ""
     }
 
-    let trackerViewID: TrackingEvent.View
+    let trackerViewID: TrackingEvent.View?
```

Comment:
Why'd this change to optional?

Reply:
Cause the new `Loading` screen doesn't have a tracker ID.

Reply:
ahh